packages/elements/src/components/ui/buttons/PrimaryButton.tsx

```diff
@@ -56,6 +56,9 @@ export const PrimaryButton = forwardRef<HTMLButtonElement, PrimaryButtonProps>(
         (loading && loadingLabel)
     );
 
+    const iconSize: ButtonSize =
+      size === "large" ? (hasLabel ? "medium" : "large") : size;
```

Comment:
Use `getIconSizeFromButtonSize`

Reply:
And should you have to change SecondayButtons and FlatButtons too?

Reply:
Secondary and Flat are just PrimaryButtons with modified styling so it should be enough to change PrimaryButton. And PrimaryButtonLink.

Reply:
But, you know, check the story to see if all buttons look good :)